config/m4/cuda.m4

```diff
@@ -41,7 +41,7 @@ AS_IF([test "x$cuda_checked" != "xyes"],
          LDFLAGS="$LDFLAGS $CUDA_LDFLAGS"
 
          # Check cuda header files
-         AC_CHECK_HEADERS([cuda.h cuda_runtime.h],
+         AC_CHECK_HEADERS([cuda.h cuda_runtime.h nvToolsExt.h],
```

Comment:
probably check it separately, and make cuda_happy=yes even if not found

Reply:
@yosefe when runtime.h is installed nvToolsExt.h is installed by default but it makes sense to separate. I'll add a separate m4 file for this.